tests/gtest/avif_fuzztest_enc_dec.cc

```diff
@@ -1,6 +1,10 @@
 // Copyright 2022 Google LLC
 // SPDX-License-Identifier: BSD-2-Clause
 
+#include <array>
+#include <cstdint>
+#include <cstring>
+#include <memory>
```

Comment:
I did not review the changes to these two fuzz tests.

Reply:
For reference the new code was simply copied from the "experimental" fuzzer tests.